.cargo/config.toml

```diff
@@ -16,6 +16,7 @@ opt-level = 1
 # This is only present for local builds, as it will be overridden
 # by the RUSTDOCFLAGS env var in CI.
 rustdocflags = ["-Arustdoc::private_intra_doc_links"]
+rustflags = ["--cfg=tokio_unstable"]
```

Comment:
I was thinking this would had gone to ci configuration only. How will developers configure mold locally if we set this here, or are these merged already?

My ~/.cargo/config.toml:

```toml
[target.x86_64-unknown-linux-gnu]
linker = "clang"
rustflags = ["-C", "link-arg=-fuse-ld=/usr/local/bin/mold"]
```

Perhaps this is just a temporary before you get to configuring the ci?

Reply:
Oh, I'm not merging this yet, it's still a draft PR ;)